code/lib/correction_functions.py

```diff
@@ -210,7 +210,7 @@ def calc_gwa_correction(paths, param):
         topo_reg = TOPO[Ind_reg]
 
         # Get the sampled frequencies from the GWA
-        w50m_gwa = pd.read_csv(paths["GWA"][:-14] + reg_name + paths["GWA"][-14:], usecols=["gwa_ws"]).to_numpy()[:, 0]
```

Comment:
can you replace it with try - except?
so that we can still use the old files

Reply:
I was thinking of just using the order of the columns, but It is possible to implement a try-expect with two different versions (first the old version, then the new version)